src/components/Synthetics/Claims/ClaimableCard.tsx

```diff
@@ -1,29 +1,62 @@
-import { t } from "@lingui/macro";
+import { Trans, t } from "@lingui/macro";
+import { PositionPriceImpactRebateInfo } from "domain/synthetics/claimHistory";
 import { MarketsInfoData, getTotalClaimableFundingUsd } from "domain/synthetics/markets";
+import { useTokensData } from "domain/synthetics/tokens";
+import { useChainId } from "lib/chains";
+import { CSSProperties, useMemo } from "react";
 import { ClaimableCardUI } from "./ClaimableCardUI";
-import { CSSProperties } from "react";
+import { calcTotalRebateUsd } from "./utils";
+import ExternalLink from "components/ExternalLink/ExternalLink";
 
 type Props = {
   onClaimClick: () => void;
+  onClaimablePositionPriceImpactFeesClick: () => void;
   marketsInfoData: MarketsInfoData | undefined;
   style?: CSSProperties;
+  claimablePositionPriceImpactFees: PositionPriceImpactRebateInfo[];
 };
 
 const tooltipText = t`Positive Funding Fees for a Position become claimable after the Position is increased, decreased or closed; or settled its fees with the option under "Accrued".`;
 const buttonText = t`Claim`;
 const title = t`Claimable`;
 
-export function ClaimableCard({ marketsInfoData, onClaimClick, style }: Props) {
+export function ClaimableCard({
+  marketsInfoData,
+  onClaimClick,
+  style,
+  claimablePositionPriceImpactFees,
+  onClaimablePositionPriceImpactFeesClick,
+}: Props) {
   const markets = Object.values(marketsInfoData ?? {});
-  const totalClaimableFundingUsd = getTotalClaimableFundingUsd(markets);
+  const totalClaimableFundingUsd = useMemo(() => getTotalClaimableFundingUsd(markets), [markets]);
+  const { chainId } = useChainId();
+  const { tokensData } = useTokensData(chainId);
+  const priceImpactRebateUsd = useMemo(
+    () => calcTotalRebateUsd(claimablePositionPriceImpactFees, tokensData, false),
+    [claimablePositionPriceImpactFees, tokensData]
+  );
 
   return (
     <ClaimableCardUI
       fundingFees={totalClaimableFundingUsd}
       buttonText={buttonText}
+      button2Text={buttonText}
+      priceImpactRebate={priceImpactRebateUsd}
       title={title}
       tooltipText={tooltipText}
+      tooltip2Text={
+        <Trans>
+          Claimable Price Impact Rebates.
+          <br />
+          <br />
+          <ExternalLink newTab href="https://docs.gmx.io/docs/trading/v2/#price-impact-rebates">
+            Read more
+          </ExternalLink>
+          .
+        </Trans>
+      }
       onButtonClick={onClaimClick}
+      onButton2Click={onClaimablePositionPriceImpactFeesClick}
       style={style}
```

Comment:
Using `button2` for text, onClick and tooltip text feels very redundant. Maybe we can use something like an array for better readability

```js
 buttons={[
    {
      text: buttonText,
      onClick: onClaimClick,
    },
    {
      text: buttonText,
      onClick: onClaimablePositionPriceImpactFeesClick,
    },
  ]}
```

Reply:
you're right, I was trying to avoid array as a prop but I guess I made a weird construction, fixed, unified ClaimableCardUI as well (will push with other changes soon)